pb_chime5/scripts/kaldi_run_rttm_libri_css.py

```diff
@@ -0,0 +1,247 @@
+"""
+[mpirun -np $(nproc --all)] python -m pb_chime5.scripts.kaldi_run_rttm_libri_css with storage_dir=<...> database_rttm=<...> [activity_rttm=<...>] [session_id=dev] [session_to_audio_paths=<...>.{yaml,json}] job_id=1 number_of_jobs=1
```

Comment:
Here you can find how to start this script.
It supports mpi parallel and kaldi parallel. Both could be used at the same time.
To disable mpi (i.e. when you have a broken installation, uninstall `mpi4py`).